notebooks/uxarray/01-Read_access_grid_data.ipynb

```diff
@@ -0,0 +1,1845 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Reading In and Accessing Grid Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook will show how to use UXarray to represent various unstructured\n",
+    "grids in the UGRID convention and to access grid information, combined with\n",
+    "Xarray to read in grid definition and data files."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dependencies\n",
+    "- uxarray\n",
+    "- xarray\n",
+    "- geocat-datafiles   (for accessing test datasets)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Background Information\n",
+    "\n",
+    "When working with unstructured grids, the grid definition and data variables\n",
+    "are often stored as separate files (most of the time as netCDF). This means\n",
+    "that there are multiple separate files that need to be read in at once.\n",
+    "\n",
+    "For example, the NOAA Geoflow project consists of 4 files (1 grid file and 3\n",
+    "data files). This project follows the UGRID conventions. Special thanks to\n",
+    "John Clyne, Shilpi Gupta, and the VAPOR team for providing these data!\n",
+    "\n",
+    "```\n",
+    "geoflow-small\n",
+    "│   grid.nc\n",
+    "│   v1.nc\n",
+    "│   v2.nc\n",
+    "│   v3.nc\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Opening The Grid and Data Files with Xarray\n",
+    "\n",
+    "First, read in the grid definition and data variable netCCDF files by using\n",
+    "`xarray.open_dataset` and/or `xr.open_mf_dataset`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
```

Comment:
Your jupyter cells are showing up with cell execution counts starting in the forties, you might want to clear/run all 
<img width="845" alt="image" src="https://user-images.githubusercontent.com/38434768/199359663-367db971-62d1-4dfa-9518-1f4a4d791a90.png">

Reply:
Thanks for catching this! Done.